v1.1/admin-ui-access-and-navigate.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Access and Navigate CockroachDB Admin UI
+toc: false
+---
+
+<div id="toc"></div>
+
+## Access the Admin UI
+
+You can access the Admin UI from any node in the cluster.
+
+By default, you can access it via HTTP on port `8080` of the hostname or IP address you configured using the --host flag while [starting the node](https://www.cockroachlabs.com/docs/stable/start-a-node.html#general). For example, `http://<any node host>:8080`. If you are running a secure cluster, use `https://<any node host>:8080`.
+
+You can also set the CockroachDB Admin UI to a custom port using `--http-port` or a custom hostname using `--http-host` when [starting each node](start-a-node.html) (i.e., each node's values are dependent on how it's started; there is no cluster-level configuration for non-default values). For example, if you set both a custom port and hostname, `http://<http-host value>:<http-port value>`. For a secure cluster, `https://<http-host value>:<http-port value>`.
+
+For additional guidance on accessing the Admin UI in the context of cluster deployment, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).
+
+## Navigate the Admin UI
+
+The Admin UI is divided into three areas:
+
+- The left-hand navigation bar that allows you to navigate to the **[Cluster Metrics dashboards](admin-ui-overview.html)**, **[Database page](admin-ui-databases-page.html)**, and **[Jobs page](admin-ui-jobs-page.html)**.
+- The main panel that shows the **[Time Series graphs](admin-ui-access-and-navigate.html#time-series-graphs)**.
+- The right-hand side panel that shows the **[Summary panel](admin-ui-access-and-navigate.html#summary-panel)** and **[Events list](admin-ui-access-and-navigate.html#events-list)**.
+
+<img src="{{ 'images/admin_ui_overview.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+### Time Series Graphs
+
+The Admin UI displays time series graphs of key metrics. Time series graphs are useful to visualize and monitor data trends. You can hover over each graph to see actual point-in-time values. 
```

Comment:
@mrtracy, should/could we say anything about the frequency at which time series data is updated in the admin UI?